synapseclient/table.py

```diff
@@ -449,7 +449,7 @@ def cast_values(values, headers):
         ## convert field to column type
         if field is None or field=='':
             result.append(None)
-        elif columnType in {'STRING', 'ENTITYID', 'FILEHANDLEID', 'LARGETEXT', 'USERID'}:
```

Comment:
Please also modify the test to include this case:  test_cast_values() in tests/unit/unit_test_tables.py